source/getting-started/enterprise-roll-out-checklist.rst

```diff
@@ -0,0 +1,278 @@
+  
+- Create a Project Charter to document goals, tasks, deliverables, and decisions 
+ - Get buy-in from project team members and key stakeholders on the project charter 
+
+Validate Essential Security and Compliance Requirements
+````````````````````````````````````````````````````````
+- Review Mattermost security features
+ - Resource: https://docs.mattermost.com/overview/security.html#security-features
+ 
+- Determine monitoring requirements
+ - Database, network, storage, log integrity, etc. 
+ - Identify fields for log management tools (e.g. Splunk Enterprise event data)
+
+- Determine environment access policies
+ - Network access, physical access, group controlled access
+
+- Determine encryption policies
+ - Resource: https://docs.mattermost.com/administration/encryption.html
+ - Resource: https://docs.mattermost.com/install/transport-encryption/config.html
+
+- Determine System Administration access policies
+ - Identify list of users or groups who need administrative access for Mattermost System Console, Command Line Tools and API privileges
+
+- Define and configure authentication policies
+ - Resource: https://docs.mattermost.com/overview/auth.html 
+ - Determine requirements for multi-factor authentication
+ - Resource: https://docs.mattermost.com/deployment/auth.html
+
+- Configure and test SSO or Corporate Directory integration (SAML or AD/LDAP)
+ - Resource: https://docs.mattermost.com/deployment/sso-saml.html
+ - Resource: https://docs.mattermost.com/deployment/sso-ldap.html
+
+- Define your mobile usage policy
+ - Resource: https://docs.mattermost.com/mobile/mobile-overview.html 
+ - Resource: https://docs.mattermost.com/deployment/mobile-app-deployment.html
+
+Create Development, Staging, and Production Environments
+````````````````````````````````````````````````````````
+- Finalize production environment design basing hardware on expected usage and requirements for high availability
+ - Resource: https://docs.mattermost.com/overview/architecture.html
+ - Resource: https://docs.mattermost.com/deployment/deployment.html 
+ - Resource: https://docs.mattermost.com/deployment/scaling.html 
+ - Resource: https://docs.mattermost.com/deployment/cluster.html
+
+- Create development and staging environments 
+ - Recommend using to test early configurations for database, authentication, file storage, Elasticsearch, prior to setting up high availability and load balancing.  
+ - Recommend configuring staging to be an identical replication of your production environment.
+
+- Create production environment
+ - Install Mattermost
+  - Install the number of nodes based on your high availability requirements outlined in your production environment design
+  - Recommendation: Use Kubernetes and the Mattermost Operator, with external supported external database and file storage solutions.  This will also provide blue/green deployment, rolling upgrades and canary builds
```

Comment:
```suggestion
  - Recommendation: Use Kubernetes and the Mattermost Operator, with external supported external database and file storage solutions. This will also provide blue/green deployment, rolling upgrades, and canary builds
```